robot/impl/robot_reconfigure_test.go

```diff
@@ -3796,6 +3844,14 @@ func (m *mockFake) Close(ctx context.Context) error {
 		m.closedAt = m.logicalClock.Add(1)
 	}
 	m.closeCount++
+
+	if m.shouldHangOnClose {
+		timer := time.NewTimer(time.Hour)
```

Comment:
if you agree with [this comment](https://github.com/viamrobotics/rdk/pull/3715/files#r1532629480) then maybe:

```suggestion
		timer := time.NewTimer(time.Minute)
```